binding/android/README.md

```diff
@@ -0,0 +1,72 @@
+# Leopard Binding for Android
+
+## Leopard Speech-to-Text Engine
+
+Made in Vancouver, Canada by [Picovoice](https://picovoice.ai)
+
+Leopard is an on-device speech-to-text engine. Leopard is:
+
+- Private; All voice processing runs locally.
+- Accurate [[1]](https://github.com/Picovoice/speech-to-text-benchmark#results)
+- Compact and Computationally-Efficient [[1]](https://github.com/Picovoice/speech-to-text-benchmark#results)
```

Comment:
Compact and Computationally-Efficient [[1]](https://github.com/Picovoice/speech-to-text-benchmark#results)

>>>

Compact and Computationally-Efficient [[2]](https://github.com/Picovoice/speech-to-text-benchmark#rtf)